rest/taskana-rest-spring/src/main/java/pro/taskana/task/rest/assembler/AttachmentSummaryRepresentationModelAssembler.java

```diff
@@ -14,10 +18,42 @@
 public class AttachmentSummaryRepresentationModelAssembler implements
     RepresentationModelAssembler<AttachmentSummary, AttachmentSummaryRepresentationModel> {
 
+  private final TaskService taskService;
+  private final ClassificationSummaryRepresentationModelAssembler classificationSummaryAssembler;
+
+  @Autowired
+  public AttachmentSummaryRepresentationModelAssembler(TaskService taskService,
+      ClassificationSummaryRepresentationModelAssembler classificationSummaryAssembler) {
+    this.taskService = taskService;
+    this.classificationSummaryAssembler = classificationSummaryAssembler;
+  }
+
   @NonNull
   @Override
   public AttachmentSummaryRepresentationModel toModel(
-      @NonNull AttachmentSummary attachmentSummary) {
-    return new AttachmentSummaryRepresentationModel(attachmentSummary);
+      @NonNull AttachmentSummary summary) {
+    AttachmentSummaryRepresentationModel repModel = new AttachmentSummaryRepresentationModel();
+    repModel.setAttachmentId(summary.getId());
```

Comment:
In the old hateoas there was an ID property of the resouce so that we had to use an alternative.
Since the change of behaviour would affect the rest service I'd prefer to create a dedicated ticket for this issue and all entities

Reply:
[TSK-1266]